packages/components/src/internal/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -312,3 +359,69 @@ export class DefaultRelatedFolderInput extends PureComponent<AssignmentOptionsIn
         );
     }
 }
+
+export class RestrictedIssueInput extends PureComponent<RestrictedOptionsProps> {
+    render() {
+        const { model, onCheckChange} = this.props;
+
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={4}>
+                    <DomainFieldLabel
+                        label="Restrict Issue List"
+                        helpTipBody={ISSUES_LIST_RESTRICTED_TRACKER_TIP}
+                        required={false}
+                    />
+                </Col>
+                <Col xs={9} lg={8}>
+                    <input
+                        type="checkbox"
+                        name="restrictedIssueList"
+                        checked={model.restrictedIssueList}
+                        onChange={onCheckChange}
+                    />
+                </Col>
+            </Row>
+        );
+    }
+}
+
+export class RestrictedIssueGroupInput extends PureComponent<RestrictedOptionsProps, any> {
+    onChange = (name: string, formValue: any, selected: Principal, ref: any): void => {
+        const groupId = selected ? selected.userId : undefined;
+        this.props.onSelect(name, groupId);
+    };
+
+    render() {
+        const { model, coreGroups } = this.props;
+
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={4}>
+                    <DomainFieldLabel
+                        label="Additional Group with Access"
+                        helpTipBody={ISSUES_LIST_RESTRICTED_GROUP_TIP}
+                        required={false}
+                    />
+                </Col>
+                <Col xs={9} lg={8}>
+                    {!coreGroups ? (
+                        <LoadingSpinner />
+                    ) : (
+                        <SelectInput
+                            name="restrictedIssueListGroup"
+                            options={coreGroups?.toArray()}
+                            placeholder="Unassigned"
+                            inputClass="col-xs-12"
+                            valueKey="userId"
+                            labelKey="displayName"
+                            onChange={this.onChange}
+                            value={model.restrictedIssueListGroup ? model.restrictedIssueListGroup : undefined}
```

Comment:
this seems like it would be equivalent to just using `value={model.restrictedIssueListGroup`, though maybe there is a reason for the null/undefined check.

Reply:
I thought so as well, but I noticed it elsewhere in this file and thought there was a good reason for it?